presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+    hive.metastore.uri=thrift://localhost:9083
+
+Partitioned Tables
+------------------
+
+Iceberg supports partitioning by specifying transforms over the table columns.
+A partition is created for each unique tuple value produced by the transforms.
+Identity transforms are simply the column name. Other transforms are:
+
+===================================== ================================================================================================
+Transform                             Description
+===================================== ================================================================================================
+``year(ts)``                          The partition value is the integer difference in years between ``ts`` and January 1st. 1970.
+
+``month(ts)``                         The partition value is the integer difference in months between ``ts`` and January 1st. 1970.
+
+``day(ts)``                           The partition value is the integer difference in days between ``ts`` and January 1st. 1970.
+
+``hour(ts)``                          The partition value is a TIMESTAMP rounded down to the nearest hour.
+
+``bucket(x, nbuckets)``               The partition value is an integer hash of ``x``, with a value between 0 and ``nbuckets - 1`` inclusive.
+
+``truncate(s, nchars)``               The partition value is the first ``nchars`` characters of VARCHAR ``s``.
+===================================== ================================================================================================
+
+In this example, the table is partitioned by month and by a hash of the account number::
+
+    CREATE TABLE iceberg.testdb.sample_partitioned (
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR)
+    WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)'])
+
+Deletion by Partition
+---------------------
+
+For partitioned tables, the Iceberg connector supports the deletion of entire
+partitions if the WHERE clause specifies an identity transform of a partition
+column.  Given this table definition::
+
+    CREATE TABLE test_partition_deletion (
+        c integer,
+        d double)
+        WITH (partitioning = ARRAY['c'])
+
+This SQL will delete all partitions for which c is 1::
+
+    DELETE FROM test_partition_deletion
+    WHERE c = 1
+
+The current Iceberg connector is based on Iceberg v1.  Iceberg v2 is not yet
+released.  When v2 is released, it will support row-level deletion and the
+Iceberg connector will be updated to provide row-level deletion.
+
+The Iceberg connector currently supports only deletion by partition.
+This SQL below will fail because the doesn't span entire partitions::
+
+    DELETE FROM test_partition_deletion
+    WHERE c = 1 AND d > 10
+
+Rolling Back to a Previous Snapshot
+-----------------------------------
+
+Iceberg supports a "snapshot" model of data, where table snapshots are
+identified by an ``long`` snapshot_ids.
```

Comment:
Let's change this to text

> where table snapshots are identified by snapshot IDs

I don't think saying "long" or "numeric" is needed here

Reply:
Changed.